modules/nw-ingress-integrating-route-secret-certificate.adoc

```diff
@@ -0,0 +1,79 @@
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-integrating-route-secret-certificate_{context}"]
+= Securing route with external certificates in TLS secrets
+
+:FeatureName: Securing route with external certificates in TLS secrets
+include::snippets/technology-preview.adoc[]
+
+The following workflow describes how to integrate third-party certificate management solutions, using the certificate reference field outlined under API Extensions.
+
+A new `.spec.tls.externalCertificate` field has been added to the route's `.spec.tls` configuration. This field can be used to specify the name of a secret containing the certificate data.
+
+[NOTE]
+====
+This feature applies to both edge routes and re-encrypt routes.
+====
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the cluster-admin role.
+
+* You must have generated the serving certificate using third-party systems.
+
+* Ensure the referenced secret contains a valid TLS certificate and key pair in PEM format. The secret must be of type `kubernetes.io/tls` and reside in the same namespace as the route you want to secure. It should include both `tls.key` and `tls.crt` keys within the `data` or `stringData` field of the secret configuration.
+
+* The cluster should have `RouteExternalCertificate` feature gate enabled.
```

Comment:
Can you please make it the first requirement, because without it other requirements won't help?

Reply:
Done

Reply:
Should I reorder the last prereq to the first one? or both 26 and 28 in the starting?